tests/lib/nested.sh

```diff
@@ -530,8 +538,8 @@ start_nested_core_vm_unit(){
 
         if [ "$ENABLE_SECURE_BOOT" = "true" ]; then
             cp -f "/usr/share/OVMF/OVMF_VARS.$OVMF_VARS.fd" "$WORK_DIR/image/OVMF_VARS.$OVMF_VARS.fd"
-            PARAM_BIOS="-drive file=/usr/share/OVMF/OVMF_CODE.$OVMF_CODE.fd,if=pflash,format=raw,unit=0,readonly=on -drive file=$WORK_DIR/image/OVMF_VARS.$OVMF_VARS.fd,if=pflash,format=raw,unit=1"
-            PARAM_MACHINE="-machine ubuntu-q35,accel=kvm -global ICH9-LPC.disable_s3=1"
+            PARAM_BIOS="-drive file=/usr/share/OVMF/OVMF_CODE.$OVMF_CODE.fd,if=pflash,format=raw,unit=0,readonly -drive file=$WORK_DIR/image/OVMF_VARS.$OVMF_VARS.fd,if=pflash,format=raw"
+            PARAM_MACHINE="-machine pc-q35-2.10,vmport=off${ATTR_KVM} -global ICH9-LPC.disable_s3=1"
```

Comment:
I'm not sure we need `vmport=off`, last time the conclusion was it's a red herring?

Reply:
Yes, makes sense to remove it.